src/main/java/org/HdrHistogram/HistogramLogProcessor.java

```diff
@@ -133,22 +142,23 @@
 
                 System.err.println("valid arguments = " + validArgs);
 
                 System.err.println(
-                        " [-h]                        help\n" +
-                                " [-v]                        Provide verbose error output\n" +
-                                " [-csv]                      Use CSV format for output log files\n" +
-                                " [-i logFileName]            File name of Histogram Log to process (default is standard input)\n" +
-                                " [-o outputFileName]         File name to output to (default is standard output)\n" +
-                                " [-tag tag]                  The tag (default no tag) of the histogram lines to be processed\n" +
-                                " [-start rangeStartTimeSec]  The start time for the range in the file, in seconds (default 0.0)\n" +
-                                " [-end rangeEndTimeSec]      The end time for the range in the file, in seconds (default is infinite)\n" +
-                                " [-outputValueUnitRatio r]   The scaling factor by which to divide histogram recorded values units\n" +
-                                "                             in output. [default = 1000000.0 (1 msec in nsec)]\n" +
-                                " [-listtags]                 list all tags found on histogram lines the input file."
+                            " [-h]                                         help\n" +
+                            " [-v]                                         Provide verbose error output\n" +
+                            " [-csv]                                       Use CSV format for output log files\n" +
+                            " [-i logFileName]                             File name of Histogram Log to process (default is standard input)\n" +
+                            " [-o outputFileName]                          File name to output to (default is standard output)\n" +
+                            " [-tag tag]                                   The tag (default no tag) of the histogram lines to be processed\n" +
+                            " [-start rangeStartTimeSec]                   The start time for the range in the file, in seconds (default 0.0)\n" +
+                            " [-end rangeEndTimeSec]                       The end time for the range in the file, in seconds (default is infinite)\n" +
+                            " [-outputValueUnitRatio r]                    The scaling factor by which to divide histogram recorded values units\n" +
+                            "                                              in output. [default = 1000000.0 (1 msec in nsec)]\n" +
+                            " [-correctLogWithKnownCoordinatedOmission i]  If larger than 0, add auto-generated value records as appropriate\n" +
```

Comment:
@giltene  I have used `i` instead of `expectedInterval` toa void having this message too large